vulnerable_people_form/integrations/nhs_openconnect_id.py

```diff
@@ -17,8 +17,8 @@ def init_app(self, app):
             self.registration_callback_url = app.config[
                 "NHS_OIDC_REGISTRATION_CALLBACK_URL"
             ]
-            self.scopes = app.config["NHS_OIDC_SCOPES"]
-            self.vtr = app.config["NHS_OIDC_VTR"]
+            self.scopes = ["openid", "profile", "email", "phone", "profile_extended"]
```

Comment:
Why are these now hardcoded?

Reply:
These are values that won't change for the service